torch/_refs/__init__.py

```diff
@@ -3649,6 +3649,17 @@ def _reshape_view_helper(a: TensorLikeType, *shape, allow_copy: bool) -> TensorL
         else:
             return _a
 
+    if a.is_contiguous() and a.device.type != "xla":
```

Comment:
@JackCaoG , @ezyang : I tried to avoid taking the specialized path by adding the condition ```a.device.type != "xla"```. But it appears the xla test is still failing on line 3653. Is the condition incorrect?

Reply:
yea it seems right to me
```
>>> import torch
>>> import torch_xla
>>> t1 = torch.randn(3,3, device=torch_xla.device())
>>> t1.device.type
'xla'
>>> t1.device.type == "xla"
True
```

Reply:
Hmm, then how to I make sense of this XLA test failure:

https://hud.pytorch.org/pr/pytorch/pytorch/127641#25765362668